src/content/1.7/development/upgrade-module/channels.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Upgrade channels
+weight: 40
+---
+
+# Upgrade channels
+
+Different channels are available for upgrade.
+
+Depending on the channel, the archive that will be used to upgrade the shop will be different.
+
+
+### Major
+
+This channel will look at the latest version, even if it is a different major version.
+
+### Minor
+
+This channel will look at the latest version of the same major version.
+
+### RC
+
+Channel currently disabled. This was done to avoid people getting stuck on pre release software of which they cannot upgrade later.
```

Comment:
```suggestion
**Channel currently disabled**. This was done to avoid people getting stuck on pre release software of which they cannot upgrade later.
```